app/ui/src/app/integrations/detail-page/detail.component.html

```diff
@@ -180,7 +180,7 @@ <h1>
                   (click)="requestAction('delete', i)">Delete Integration</button>
         </dd>
       </dl>
-      <dl class="dl-horizontal">
+      <dl class="dl-horizontal" *ngIf="config.getSettings('features', 'logging', false)">
```

Comment:
Method calls should never be injected in HTML templates. Otherwise the Angular detection system will execute that method call every few milliseconds until the component is unmounted. You'd better execute this just once at the `OnInit` stage and make it accessible from a property in the component class.